### PR TITLE
add mappings to BondType, ChargeClass, Race

### PR DESCRIPTION
**Pull request by jamwalla**
Global maps:
- 'OTHER': ChargeClass.OTHER
- 'ASIAN/PACIFIC ISLANDER': Race.ASIAN

us_fl_osceola overrides:
- 'A': Race.ASIAN

us_fl_nassau overrides:
- 'NO PC': BondType.NO_BOND

### CODE REVIEW

**Comment by jessex** [comment]
Have we found a data dictionary / spec for that vendor?

File: recidiviz/common/constants/bond.py [hunk 1]
```diff
@@ -45,8 +45,10 @@ def _get_default_map():
 _BOND_TYPE_MAP = {
     'BOND DENIED': BondType.BOND_DENIED,
     'CASH': BondType.CASH,
+    'CASH BOND': BondType.CASH,
     'NO BOND': BondType.NO_BOND,
     'ROR': BondType.NO_BOND,
+    'NO PC': BondType.NO_BOND,
```
[the remark this answers is not in the document]
Comment (ohinds) [reply recidiviz/common/constants/bond.py]: Or at least put it in a vendor- or region-specific map instead of the global one?

**Comment by jamwalla** [comment]
note to self: add `A` == Asian to us_fl_osceola

File: recidiviz/common/constants/bond.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jamwalla) [reply recidiviz/common/constants/bond.py]: done, thanks! can you take another look?

**Comment by jamwalla** [comment]
closing this because 1) we (sort of) have a process for finding and fixing these, 2) the overrides object has changed, 3) most of this is fixed by #1022 and others.

**Pull request closed without merging**